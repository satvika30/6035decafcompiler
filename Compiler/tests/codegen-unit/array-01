class Program {
	int a[5];
	
	void main() {
		a[0] = 10;
		a[1] = a[0] + 1;
		a[2] = a[0] + a[1];
		a[3] = 9;
		a[4] = a[0] * a[1];
		
		for i = 0, 5 {
			callout("printf","%d\n", a[i]);
		}
	}
}