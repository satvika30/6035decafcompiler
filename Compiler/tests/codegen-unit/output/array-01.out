10
11
21
9
110
11
